samples/graph-notification-team-channel/csharp/README.md

```diff
@@ -0,0 +1,104 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates use of team/Channel subscription that will post notifications when user create/edit/delete.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
```

Comment:
Few fixes needed for Sample code metadata present in Readme file.  
Content Type and created date are part of extensions so we need to add one space in front of contentType and CreatedDate 

We need to start adding urlFragment in metadata. We can keep urlFragment and name present in sample.json file same.  

Refer below:
https://microsoft.sharepoint.com/teams/Teams-BLR2/_layouts/15/Doc.aspx?sourcedoc=%7Bed4de38b-a06a-4d3f-a921-854c73c24ee3%7D&action=edit&wd=target(PR%20quick%20check.one%7C69fc4337-32c5-4e46-b39e-7e4df9132652%2FSamples%20PR%20review%20checklist%7C42d47188-22e3-4272-8a86-cab06cc98de5%2F)&wdorigin=NavigationUrl&share=IgGL403taqA_TakhhUxzwk7jAfnhKCV9Li_J3933KBD4l8Y